contents/docs/integrations/js-integration.md

```diff
@@ -155,7 +155,13 @@ Using `register_once` will ensure that if a property is already set, it will not
 
 ### Opt Users Out
 
-PostHog JS offers a function to opt users out based on your cookie settings definition (e.g. preferences set via a cookie banner). You can:
+PostHog JS offers a function to opt users out based on your cookie settings definition (e.g. preferences set via a cookie banner). 
+
+This is also the suggested way to prevent capturing data from the admin on the page, as well as from team members of your organization. A simple way to do this is to access the page as the admin (or any other user on your team you wish to stop capturing data on), and call `posthog.opt_out_capturing();` on the developer console. You can also add this logic in you app and call it directly after an admin/team member logs in. 
```

Comment:
Genuine question - why is this suggested as better than just using cohorts to ignore traffic?

Also, does this call persist with the cookie?

Most people asking for how to ignore their own team's traffic have ie developers working locally that refresh pages a lot to test out changes. If it *doesn't* persist, then it feels more logical to use cohorts so you don't have to suddenly make this call over and over again. If it *does* persist, wouldn't that mean you could trip yourself up later if you were trying to check if PostHog is working properly?

Reply:
Right, so this was originally suggested by Marius on https://github.com/PostHog/posthog.com/issues/297. The talk was about admins rather than team members, so perhaps it makes more sense in that context. 

Either way, it does persist. I think using this method comes down to the extent you want to ignore data.

If you don't want to ever get data on some users, use this. If you want to mostly not gather data on some users, this could be for you or not. If you want to most data by a user, but just exclude some things, then you should look into something else. 

The reason I defaulted to this from your comment @jamesefhawkins is because the issue mentioned **ignoring** traffic, which is what this does. Cohorts don't ignore anything, they just let you differentiate between sets of data. 

Nevertheless, the solution for the Docs here should be as follows: I'll add a note about this on Cohorts, and cross-link between this method and the specific section in cohorts. This way users will be able to always find the alternatives and make the decision that makes sense for them.

Reply:
Also, another reason I like this is that just pasting it on the dev console will work. So if you're having a session you want to completely ignore, then without adding logic to your app you can do `posthog.opt_out_capturing()` at the start and `posthog.opt_in_capturing()` at the end.